src/map/skill.cpp

```diff
@@ -13171,9 +13171,10 @@ int32 skill_castend_nodamage_id (struct block_list *src, struct block_list *bl,
 			}
 			heal = heal * (100 + status_get_crt(src)) * status_get_lv(src) / 10000;
 			status_heal(bl, heal, 0, 0, 0);
-			clif_skill_nodamage(src, *bl, skill_id, heal);
+			clif_skill_nodamage(src, *bl, AL_HEAL, heal);
```

Comment:
```suggestion
			clif_skill_nodamage(0, *bl, AL_HEAL, heal);
```
to not display "Heal" text

Reply:
Edited your comment from 0 to nullptr.